mbf_abstract_nav/include/mbf_abstract_nav/abstract_planner_execution.h

```diff
@@ -41,16 +41,24 @@
 #ifndef MBF_ABSTRACT_NAV__ABSTRACT_PLANNER_EXECUTION_H_
 #define MBF_ABSTRACT_NAV__ABSTRACT_PLANNER_EXECUTION_H_
 
-#include <pluginlib/class_loader.h>
-#include <boost/chrono/thread_clock.hpp>
+#include <map>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
 #include <boost/chrono/duration.hpp>
-#include <tf/transform_listener.h>
+#include <boost/chrono/thread_clock.hpp>
+#include <boost/shared_ptr.hpp>
+#include <boost/thread.hpp>
+
 #include <geometry_msgs/PoseStamped.h>
+#include <pluginlib/class_loader.h>
```

Comment:
I think we do not need the class_loader in mbf_abstract_nav anymore. Or?

Reply:
agree